combat/Combat.gd

```diff
@@ -58,11 +59,11 @@ func _ready():
 	add_combatant(create_combatant(CombatantDatabase.combatants["goblin"], "Goblin 2"), 1, Vector2i(10,7))
 	add_combatant(create_combatant(CombatantDatabase.combatants["goblin"], "Goblin 3"), 1, Vector2i(10,9))
 	
-	emit_signal("update_turn_queue", combatants, turn_queue)
+	emit_signal("update_turn_queue", combatants, turn_queue) # connected to UI
 	
 	controller.set_controlled_combatant(combatants[turn_queue[0]])
 	game_ui.set_skill_list(combatants[turn_queue[0]].skill_list)
-
+	camera.position = Vector2(combatants[current_combatant].position * 32.0) + Vector2(16, 16)
```

Comment:
Code interfacting with the Camera should be separate from the Combat class. The Combat class should instead use signals to inform the Camera class that it should change its position.